documentation/writing_tests.md

```diff
@@ -16,4 +16,21 @@
 
 ## Structure of tests
 
+a. Internal tests.
```

Comment:
do we want to refer to these as `unit` vs `integration` tests, since that's what package.json tasks refer to them as?

Reply:
I hesitate from making that distinction since these words convey more meaning than the one we intend to portray. I would like more people to chime in on this discussion.

Reply:
I have updated this file with something more concrete, after speaking with Brian Terlson and Richard Park.

Reply:
I find `unit` vs `integration` to be a fuzzy line, especially for libraries, and npm tasks are easy to change. Let's align on words that have concrete meaning for us! I like the current proposal.

Reply:
Yeah I'm good with this. We should change the task names to reflect this after we finalize these docs!